README.md

```diff
@@ -12,7 +12,8 @@
 </h4>
 
 <p align="center">
-  <a href="#packages">Packages</a> ∙
+  <a href="#packages">Packages</a>

```

Comment:
I think you removed the separator on accident? `∙` Should add one after `Architecture` too

Reply:
Is that what that was? I thought it was a typo. 🤷‍♂️ I'll add it back. Good catch